endpoint/templates/tests.php

```diff
@@ -31,6 +42,70 @@ function test_class_access() {
 	 * @since  <%= version %>
 	 */
 	function test_sample() {
+		/*
```

Comment:
Sorry for such a lengthy delay on looking at this PR.

Can you please remove the commented code and I'll get this merged?